tools/dataset.h

```diff
@@ -1,100 +0,0 @@
-
-#pragma once
-
-#ifdef __cplusplus
```

Comment:
Why did this file move to src? And why is it marked as deleted instead of moved?

Reply:
It's fixed.